### PR TITLE
Nightwatch Test Cases Added.

### PR DESCRIPTION
**Pull request by JadhavNamisha**

### CODE REVIEW

File: nightwatch/README.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ npx nightwatch </path/of/the/test/case>
 
 Path can be individual test case or a directory which contains multiple test cases.
 
+In case you using ddev and running custom commands, please find below command to execute the test case:
+ddev ssh
```
Comment (pranitjha) [inline nightwatch/README.md]: remove this line.

File: nightwatch/tests/CreateUser.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  loginHelper.login(browser);
+
+  browser
+  .navigateTo(browser.globals.application_url + '/admin/people')
+  .click("#block-claro-local-actions a")
+  .click("#edit-mail")
+  .setValue("#edit-mail", "test@test.com")
+  .click("div.page-wrapper")
+  .perform(function() {
+            const actions = this.actions({async: true});
```
Comment (pranitjha) [inline nightwatch/tests/CreateUser.js]: Use 2 space indentation

**Review by pranitjha — CHANGES_REQUESTED** [review]
Remove the .DS_Store files from the PR.

File: nightwatch/README.md (hunk 1 above)
Comment (JadhavNamisha) [reply nightwatch/README.md]: Removed the line.

**Pull request merged**